cinema/models.py

```diff
@@ -126,5 +132,5 @@ def __str__(self):
                 f"(row: {self.row}, seat: {self.seat})")
 
     class Meta:
-        unique_together = ("movie_session", "row", "seat")
+        unique_together = ["movie_session", "row", "seat"]
```

Comment:
The `unique_together` option is deprecated in favor of `UniqueConstraint` with the `constraints` option in the `Meta` class. Consider updating this to use `UniqueConstraint` for future compatibility.